doc/sphinx/changes.rst

```diff
@@ -8,6 +8,694 @@ Recent changes
+  (`#13955 <https://github.com/coq/coq/pull/13955>`_,
+  by Andrej Dudenhefner, with help from Olivier Laurent).
+- **Added:**
+  ``Cantor.v`` containing the Cantor pairing function and its inverse.
+  ``Cantor.to_nat : nat * nat -> nat`` and ``Cantor.of_nat : nat -> nat * nat``
+  are the respective bijections between ``nat * nat`` and ``nat``.
+  (`#14008 <https://github.com/coq/coq/pull/14008>`_,
+  by Andrej Dudenhefner).
+- **Added:**
+  Lemmas to ``Q``: ``Qeq_from_parts``, ``Qden_cancel``, ``Qnum_cancel``, ``Qreduce_l``, ``Qreduce_r``, ``Qmult_inject_Z_l``, ``Qmult_inject_Z_r`` QArith_base
+  Reduction of rationals; establishing equality for Qden/Qnum separately.
+  (`#14087 <https://github.com/coq/coq/pull/14087>`_,
+  by Karolin Varner).
+- **Added:**
+  Added ``Coq.Structures.OrdersEx.String_as_OT`` and
+  ``Coq.Structures.OrdersEx.Ascii_as_OT`` to make strings and ascii ordered
+  types (using lexical order).  (`#14096
+  <https://github.com/coq/coq/pull/14096>`_, by Jason Gross).
+- **Added:**
+  Lemmas :g:`app_eq_app`, :g:`Forall_nil_iff`, :g:`Forall_cons_iff` to ``List.v``
+  (`#14153 <https://github.com/coq/coq/pull/14153>`_,
+  closes `#1803 <https://github.com/coq/coq/issues/1803>`_,
+  by Andrej Dudenhefner, with help from Olivier Laurent).
+- **Added:**
+  ``Z``, ``positive`` and ``N`` constants can now be printed in hexadecimal by
+  opening ``hex_Z_scope``, ``hex_positive_scope``, and ``hex_N_scope``
+  respectively (`#14263 <https://github.com/coq/coq/pull/14263>`_, by Jason
+  Gross).
+- **Added:**
+  Absolute value function for Sint63.
+  (`#14384 <https://github.com/coq/coq/pull/14384>`_,
+  by Ana Borges).
+- **Added:**
+  Lemmas showing :g:`firstn` and :g:`skipn` commute with :g:`map`.
+  (`#14406 <https://github.com/coq/coq/pull/14406>`_,
+  by Rudy Peterson).
+- **Fixed:**
+  Haskell extraction is now compatible with GHC versions >= 9.0.  Some ``#if``
+  statements have been added to extract ``unsafeCoerce`` to its new location in
+  newer versions of GHC.  (`#14345 <https://github.com/coq/coq/pull/14345>`_,
+  fixes `#14256 <https://github.com/coq/coq/issues/14256>`_, by Jason Gross).
+
+Infrastructure and dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Coq's configure script now requires absolute paths for the `-prefix`
+  option.
+  (`#12567 <https://github.com/coq/coq/pull/12567>`_,
+  by Emilio Jesus Gallego Arias).
+- **Changed:**
+  The regular Coq package has been split in two: coq-core, with
+  OCaml-based libraries and tools; and coq-stdlib, which contains the
+  Gallina-based standard library. The package Coq now depends on both
+  for compatiblity.
+  (`#12567 <https://github.com/coq/coq/pull/12567>`_,
+  by Emilio Jesus Gallego Arias, review by Vincent Laporte, Guillaume
+  Melquiond, Enrico Tassi, and Théo Zimmerman).
```

Comment:
In practice, this change may end up being limited to the opam packages in the git repository for 8.14, given that only the full Dune mode actually supports this split... Maybe this should be clarified.

Reply:
Shouldn't we just revert the commit on the branch then?

Reply:
Unfortunately, that's not possible because the Dune build uses this package separation (so that `core-core` is the part that is always built with Dune now).